app/models/visitors_step.rb

```diff
@@ -79,9 +84,41 @@ def validate_email
     end
   end
 
+  # rubocop:disable Metrics/AbcSize
+  # rubocop:disable Metrics/MethodLength
   def validate_visitors
-    ages = visitors.map(&:age).compact
-    visitor_constraints.validate_visitor_ages_on self, :general, ages
-    visitor_constraints.validate_visitor_number self, :general, visitors.size
+    # It's invalid if there are no visitors, but there's no need to call the API
+    if visitors.empty?
```

Comment:
I'd prefer if this was handled by the AP as it is a validation on the group of visitors, what do you think?